tools/cpp/crosstool_lib.bzl

```diff
@@ -284,7 +284,7 @@ def _is_linux(platform):
     return platform == "k8"
 
 def _is_darwin(platform):
-    return platform == "darwin"
+    return platform.startswith("darwin")
```

Comment:
I might be able to change this to just `darwin_x86_64`, I'm not entirely sure, but also if we eventually have to support a new `darwin`, I assume this is what we'll want